Automattic/wp-calypso
# /read/susbcriptions/: Add unsubscribed feeds search results to subscribed-to sites search result

![uvpBcE.gif](https://github.com/Automattic/wp-calypso/assets/2019970/6a6ed6d8-5550-4286-bbe5-d84449879e3e)


Fixes https://github.com/Automattic/wp-calypso/issues/78631

Related designs: inDLaEQV8jJ21O4WXawIsJ-fi-3741_37507

## Proposed Changes

* When user searches through their site subscriptions, use the search query to search for unsubscribed sites at the same time, and display the results at the bottom of the subscribed-to sites
* There is some ongoing discussion in relation to copies that should be used to label the additional results. For now, I used: 
`No results found for “%s”. Here are some other sites that match your search.`, and ` Here are some other sites that match your search.`. Related conversation: p1688572932711179-slack-C02TCEHP3HA
* Loading placeholders for the additional results are still to be implemented
* After subscribing, it takes a while for the backend elastic search to take that in, so for a few minutes after we subscribe to a site, it still comes back in the search results, even despite the fact that `exclude_followed` URL query flag is set to true. As a (short-term?) solution, I amend the copy of the “Subscribe” button to “Subscribed” and make it disabled. 
   cc @saygunnyc
   ![Screenshot 2023-07-05 at 23 23 13](https://github.com/Automattic/wp-calypso/assets/2019970/5260d72a-7e8e-4ebb-b6a0-af18ec2c4830)
 

## Testing Instructions



* Go to http://calypso.localhost:3000/read/subscriptions
* Play around with the search input
* Ensure you get results with unsubscribed sites at the bottom, when a search text is entered
* Ensure the additional results are not displayed when the search is cleared 

## Pre-merge Checklist
